packages/api-headless-cms/__tests__/contentAPI/contentModelToSDL.read.test.ts

```diff
@@ -5,9 +5,12 @@ import contentModels from "./mocks/contentModels";
 import categorySDL from "./snapshots/category.read";
 import productSDL from "./snapshots/product.read";
 import reviewSDL from "./snapshots/review.read";
+import { CmsModelFieldToGraphQLPlugin } from "~/types";
 
 describe("READ - ContentModel to SDL", () => {
-    const fieldTypePlugins = graphqlFieldPlugins().reduce((acc, pl) => {
+    const fieldTypePlugins = graphqlFieldPlugins().reduce<
+        Record<string, CmsModelFieldToGraphQLPlugin>
+    >((acc, pl) => {
```

Comment:
is `pl` a standard definition we use elsewhere? Otherwise would it make it more self documenting to use the full name even though its more verbose?

Reply:
We don't have a standard for variable names at all 😅

Reply:
No worries maybe something we could look at at some point, but I do think it's worth being more verbose for the sake of clarity.

Reply:
It's a good point though, I'll keep that in mind for all the future code I'm writing. I think it has more to do with the laziness than any particular style :D